src/models/product.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Product = sequelize.define('Product', {
+    color: DataTypes.STRING,
+    isFavorite: DataTypes.BOOLEAN,
+    name: DataTypes.STRING,
+    reviews: DataTypes.INTEGER
```

Comment:
It should be a separate table.

Reply:
For good yes, but according to task description we should create tables only for ​users​​ and ​products​​.

Reply:
https://epa.ms/nodejs18-hw6